benchmark/src/main/scala/spire/benchmark/ComplexAddBenchmarks.scala

```diff
@@ -15,27 +15,27 @@
 
 package spire
 package benchmark.jmh
```

Comment:
Thank you and sorry for my late response😖 I will check it!

Reply:
> java.lang.ClassCastException: class spire.math.Complex$mcD$sp cannot be cast to class java.lang.Comparable 

This was because `Complex` does not extend `Ordered` and we need to use `FakeComplex` defined at the begining of the file. I was mistakenly using `cs` instead of `cs2` 😖